### PR TITLE
Remove com.waz.api.AccentColor and the old AccentColorController

### PR DESCRIPTION
**Pull request by makingthematrix**
I found one more place, in `DrawingFragment` where a mixup of `com.waz.api.AccentColor` and `com.waz.api.impl.AccentColor` caused a crash on opening. I looked through the rest of UI and replaced `import com.waz.api._` in a few places with what was actually used in the given file. It's safer this way.

Requires https://github.com/wireapp/wire-android-sync-engine/pull/396
#### APK
[Download build #11416](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11416/artifact/build/artifact/wire-dev-PR1705-11416.apk)
[Download build #11421](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11421/artifact/build/artifact/wire-dev-PR1705-11421.apk)
[Download build #11422](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11422/artifact/build/artifact/wire-dev-PR1705-11422.apk)
[Download build #11428](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11428/artifact/build/artifact/wire-dev-PR1705-11428.apk)
[Download build #11430](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11430/artifact/build/artifact/wire-dev-PR1705-11430.apk)
[Download build #11433](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11433/artifact/build/artifact/wire-dev-PR1705-11433.apk)

### CODE REVIEW

**Comment by AndroidBob** [comment]
Can one of the admins verify this patch?

File: app/src/main/java/com/waz/zclient/markdown/MarkdownTextView.java [hunk 1]
```diff
@@ -84,9 +83,9 @@ private void configureStyleSheet() {
         // update the link color whenever the accent color changes
         ((BaseActivity) getContext()).injectJava(AccentColorController.class).accentColorForJava(new AccentColorCallback() {
             @Override
-            public void color(AccentColor color) {
-                mStyleSheet.setLinkColor(color.getColor());
-                setLinkTextColor(color.getColor());
+            public void color(com.waz.model.AccentColor color) {
```
Comment (deanrobertcook) [inline app/src/main/java/com/waz/zclient/markdown/MarkdownTextView.java]: fix import

**Comment by JPonte** [comment]
build again

**Comment by makingthematrix** [comment]
build again

**Comment by makingthematrix** [comment]
build again

**Pull request merged**